### PR TITLE
Code for "Data augmentation strategies to improve reaction yield pred…

### PR DESCRIPTION
**Pull request by pschwllr**
…ictions and estimate uncertainty", Machine Learning for Molecules Workshop @ NeurIPS 2020

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,13 +57,52 @@ We explored two high-throughput experiment (HTE) data sets and then also the yie
...
+
+#### Test-time-augmentation
+
+
+<div style="text-align: center">
+<img src="nbs/images/testtime_augmentation.png" width="800">
+<p style="text-align: center;"> <b>Figure:</b> Test-time augmetation.</p>
+</div>
+
+A typical non-bayesian way of estimating epistemic uncertainty, is to take predictions of models trained with a different seed. Here, we explore different strategy which requires a single model, namely, test-time augmentation. At test-time, we generate multiple augmented versions of the same input and then, average the prediction of the same model and take the standard deviation as uncertainty estimate.
```
Comment (avaucher) [inline README.md]: "we explore *a* different strategy"?

File: README.md [hunk 2]
```diff
@@ -100,7 +139,8 @@ We performed different experiments using random and time splits on the reaction
 
 ## Citation
 
-If you found this repo useful, please cite our [manuscript](https://chemrxiv.org/articles/preprint/Prediction_of_Chemical_Reaction_Yields_using_Deep_Learning/12758474).
+If you found this repo useful, please cite our the following preprints:
```
Comment (avaucher) [inline README.md]: "cite the following preprints" (delete "our")

**Review by avaucher — COMMENTED** [review]
Looks good to me, I only spotted two typos in the README.

File: README.md (hunk 1 above)
Comment (pschwllr) [reply README.md]: Good catch!

File: README.md (hunk 2 above)
Comment (pschwllr) [reply README.md]: Thanks!

**Review by avaucher — APPROVED** [review]

**Pull request merged**